src/Factory/AttributePipelineFactory.php

```diff
@@ -20,7 +19,6 @@ public function create(): PipelineInterface
         return $pipeline
             ->pipe($this->taskProvider->get(RetrieveAttributesTask::class))
             ->pipe($this->taskProvider->get(CreateUpdateEntityTask::class))
-            ->pipe($this->taskProvider->get(DeleteEntityTask::class))
```

Comment:
Why we remove it ?

Reply:
we don't remove attributes by default because it is too dangerous but the task is still available if needed